### DISCUSSION TITLE
What affects the malware graph?

### DISCUSSION THREAD

**Discussion by agneevX** [AdguardTeam/AdGuardHome#4081] [Q&A]
This graph is always nil:

<img width="257" alt="Screen Shot 2022-01-04 at 1 16 00 PM" src="https://user-images.githubusercontent.com/19761269/148025996-342023d7-8230-400a-a68e-9fdd9427d35c.png">

What affects this specifically?

**Comment by ainar-g** [comment]
That's the graph of the requests for which Safe Browsing was triggered.  That is, malware sites, phishing websites, and so on.

**Reply by agneevX** [reply]
Does this rely on blocklists?

**Reply by ainar-g** [reply]
No, it uses domain hash prefixes (2 bytes) which are checked against the database on the AdGuard servers.  See https://kb.adguard.com/en/general/how-malware-protection-works.

**Reply by agneevX** [reply]
Are you referring to this?

![Screenshot (84)](https://user-images.githubusercontent.com/19761269/148071384-69bec087-90e8-4623-88f6-529296eaafbb.png)


This is quite confusing because this makes no mention of malware.

> That's the graph of the requests for which Safe Browsing was triggered.

Hmm. They're completely different things though.

**Reply by ainar-g** [reply]
I agree that there is a degree of confusion in terms here.  I'll talk to the UI people when they're back from the winter holidays.

**Reply by agneevX** [reply]
Then there's malware blocklists, that don't seem to be related to the graph??

![Screenshot (85)](https://user-images.githubusercontent.com/19761269/148071618-de628010-ceb2-460f-a289-c15c90501530.png)

**Reply by ainar-g** [reply]
@agneevX, I don't quite understand your question.  AGH does not currently use the filtering list category in any meaningful way, if that's what you're asking.  It's just a visual grouping, at least currently.  That may change in the future.
